tests/translate/translate_updatedzc.py

```diff
@@ -5,11 +5,11 @@
 class TranslateUpdateDzC(TranslateFortranData2Py):
     def __init__(self, grid):
         super().__init__(grid)
-        update_gz_on_c_grid = updatedzc.UpdateGeopotentialHeightOnCGrid(grid)
+        self.update_gz_on_c_grid = updatedzc.UpdateGeopotentialHeightOnCGrid(grid)
 
         def compute(**kwargs):
             kwargs["dt"] = kwargs.pop("dt2")
-            update_gz_on_c_grid(**kwargs)
+            self.update_gz_on_c_grid(**kwargs)
```

Comment:
This change is not necessary, can you revert it? `compute` both before and after is referencing a variable from the outer scope of the function, the variable being referenced has just been changed from `update_gz_on_c_grid` to `self` (since `self` is undefined in the local scope of this `def compute`).